testing/rabbitmq-c/APKBUILD

```diff
@@ -1,27 +1,50 @@
 # Contributor: Fabio Ribeiro <fabiorphp@gmail.com>
 # Maintainer: Fabio Ribeiro <fabiorphp@gmail.com>
 pkgname=rabbitmq-c
-pkgver=0.7.1
-pkgrel=0
+pkgver=0.8.0
+pkgrel=1
 pkgdesc="RabbitMQ C client"
 url="https://github.com/alanxz/rabbitmq-c"
+subpackages="$pkgname-bin $pkgname-doc $pkgname-dev"
+depends_dev="openssl-dev popt-dev"
+makedepends="$depends_dev cmake xmlto doxygen"
 arch="all"
 license="MIT"
-source="https://github.com/alanxz/$pkgname/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/alanxz/$pkgname/archive/v$pkgver.tar.gz"
 
 builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
 	cd "$builddir"
-	./configure --prefix=/usr || return 1
+	rm -rf build && mkdir build && cd build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr ..
 	make || return 1
 }
 
 package() {
-	cd "$builddir"
+	cd "$builddir"/build
 	make DESTDIR="$pkgdir" install || return 1
+	cd "$pkgdir"/usr
+	if [ -d ./lib64 ]; then
+		mv lib64 lib
```

Comment:
This installs to lib64 on x86_64 builds, which doesn't appear to be an appropriate location on alpine.